app/src/db/migrations/20200521134300_007-settings-receipt-to-submission.js

```diff
@@ -0,0 +1,37 @@
+const PREFIX = require('../../forms/minesoperatorscreening/constants').PREFIX;
+
+exports.up = function (knex) {
+  return Promise.resolve()
+    .then(() => {
+      const config = {
+        template: 'confirmation-number-email.html',
+        from: 'MCAD.RegionalOps@gov.bc.ca',
+        subject: 'Industrial Camps Form Submission',
+        title: 'Industrial Camps Form Submission',
+        priority: 'normal',
+        messageLinkText: 'Click to view your Submission',
+        messageLinkUrl: 'https://comfort-dev.pathfinder.gov.bc.ca/app/minesoperatorscreening/review'
```

Comment:
Reminder we'll need to update the DB in test and prod after the migrations to set these appropriately right?

Reply:
Correct.